.github/workflows/solutions-dpg-qa.yaml

```
@@ -0,0 +1,156 @@
+          sudo mv sops-v3.7.1.linux /usr/local/bin/sops
+
+      - name: sops decryption
+        run: |
+          sudo sops --decrypt --age ${{ secrets.PUBLIC_AGE_KEY }} $HOME/work/pucar-DevOps/pucar-DevOps/deploy-as-code/charts/environments/solutions-dev-to-qa-secrets.yaml > env-secrets1.yaml
+          cp env-secrets1.yaml $HOME/work/pucar-DevOps/pucar-DevOps/deploy-as-code/charts/environments/solutions-dev-to-qa-secrets.yaml
+      #        working-directory: deploy-as-code/charts/environments
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: 'latest' # Specify the version of Helm
```

yamllint

[warning] too few spaces before comment

(comments)